### ISSUE TITLE
Incorrect sniff name after scan

### ISSUE THREAD

**Issue by roslov** [matthewbdaly/abstract-or-final-sniff#2]
When I run `phpcs --standard=vendor/matthewbdaly/abstract-or-final-sniff/src/AbstractOrFinalClassesOnly/ruleset.xml -p -s src/Entity/CampaignStatus.php` I receive such an error:

```
FILE: /app/src/Entity/CampaignStatus.php
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 40 | ERROR | [x] All classes should be declared using either the "abstract" or "final" keyword
    |       |     (Matthewbdaly.Sniffs.AbstractOrFinal.Matthewbdaly\AbstractOrFinalClassesOnly\Sniffs\AbstractOrFinalSniff)
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

What should I change to have the sniff name like `AbstractOrFinalClassesOnly` but not `Matthewbdaly.Sniffs.AbstractOrFinal.Matthewbdaly\AbstractOrFinalClassesOnly\Sniffs\AbstractOrFinalSniff`?

**Comment by matthewbdaly** [comment]
Sorry, I don't know. Codesniffer is rather arcane in that regard, and the documentation is rather hard to follow. Given that:

- This was a single rule I wrote to fix one single issue that bothered me
- It was never planned to be part of a large, more complex coding standard
- I'd managed to get it working, which was all I really needed - I use `dealerdirect/phpcodesniffer-composer-installer` to do the heavy lifting

Then I never really polished it up further than that. To date this is my only Codesniffer rule I've written, so it wasn't really worthwhile digging further to sort out something like that.

If it was being made part of a large coding standard, then it would probably be straightforward to just drop it into one of the existing folders and amend the namespace accordingly, and it'd just work.
